docs-translations/zh-CN/api/notification.md

```diff
@@ -0,0 +1,109 @@
+# 通知
+
+> 创建系统桌面通知
+
+进程: [Main](../glossary.md#main-process)
+
+## 在渲染器进程中使用
+
+如果你想要从渲染器进程显示通知，则应使用 [HTML5 Notification API](../tutorial/notifications.md)
```

Comment:
Page not exist

Reply:
https://github.com/electron/electron/pull/10748